internal/uri.go

```diff
@@ -0,0 +1,45 @@
+package internal
+
+import (
+	"net/url"
+	"os"
+	"strings"
+)
+
+// https://buildpacks.io/docs/reference/config/package-config/#dependencies-list-optional
```

Comment:
Can you please remove this comment, it does not really explain what this function does.